client/my-sites/plans-features-main/plan-type-selector.tsx

```diff
@@ -137,6 +139,8 @@ export const IntervalTypeToggle: React.FunctionComponent< IntervalTypeProps > =
 		return null;
 	}
 
+	const redirectTo = props.redirectTo ? { redirect_to: props.redirectTo } : {};
```

Comment:
I'd say the naming of that constant is a bit confusing. `props.redirectTo` comes in as a `string | null | undefined`, and then we name a variable the same way, but it's an object. I think this can be resolved by renaming the constant to something else, like `redirectProps` or `additionalPathProps` for example.